errors/Readme.md

```diff
@@ -190,441 +155,12 @@ If we encounter a Rust panic, that is covered in the [Unknown Errors](#unknown-e
 
 SDK acts as the interface between the binaries and the tools. This section covers errors from SDK, binaries and the network between SDK ⇆ Binary and Binary ⇆ Data source.
 
-### Common
+This spec does not list the errors. Instead the single source of truth for them is our code. Here is a list to their defintions for each error category:
```

Comment:
Maybe note here that the error codes can now be seen after clicking the rust links.